vars/withInfraPipeline.groovy

```diff
@@ -42,6 +44,7 @@ def call(String product, String component = null, Closure body) {
         sectionInfraBuild(
           subscription: subscription.nonProdName,
           environment: environment.nonProdName,
+          aksSubscription: aksSubscriptions.sandbox,
```

Comment:
isn't this unused?

Reply:
This is what the shared-infra pipelines are using initially? https://github.com/hmcts/cnp-plum-shared-infrastructure/blob/master/Jenkinsfile_pipeline_test#L14

Reply:
If it's not added here the env checking wouldn't run - which we want to happen during this build as far as I understood?

Reply:
Actually I suppose we only need this before a `Deploy to SBOX` stage right? in which case this can probably go, as you say @timja